### ISSUE TITLE
Guided Transfer: Combine GuidedTransferOptions and GuidedTransferDetails into a single component

### ISSUE THREAD

**Issue by jordwest** [Automattic/wp-calypso#6166]
`<GuidedTransferOptions/>` and `<GuidedTransferDetails/>` don't need to be separate components. The `GuidedTransferDetails` component can be merged into `GuidedTransferOptions` after #6048 and #6015 have landed.

**Comment by jordwest** [comment]
Done in #7347
